clair/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Add a `namespace` field please.

Reply:
What should be the value?

Reply:
`namespace: {{ .Release.Namespace }}`

And you pass that when you use helm, e.g. `helm template . --name your-release-name --namespace your-namespace`